page/ts/index.ts

```diff
@@ -30,10 +25,25 @@ const getParticipateModal: any = (text: string) => {
       </div>`,
   };
 };
-$(document).ready(() => {
+
+const movePlatformButtonToAnotherColumn = () => {
```

Comment:
please check if it's still used (i am pretty sure this button is removed) and remove whole function then